model/lifecycle.go

```diff
@@ -1442,7 +1446,20 @@ func AddNewTasks(ctx context.Context, activated bool, v *Version, p *Project, pa
 		// a record in the TVPairSet indicating that it should exist
 		tasksToAdd := []string{}
 		for _, taskname := range pairs.ExecTasks.TaskNames(b.BuildVariant) {
-			if _, ok := existingTasksIndex[taskname]; ok {
+			if info, ok := existingTasksIndex[taskname]; ok {
+				if !info.activated && activated { // update task activation for dependencies that already exist
+					var fullTask *task.Task
+					fullTask, err = task.FindOneId(info.id)
+					if err != nil {
+						return nil, err
+					}
+					if fullTask == nil {
+						return nil, errors.Errorf("task '%s' not found", info.id)
```

Comment:
Use Wrapf

Reply:
Here Wrapf doesn't apply because there is no error